src/Chirp.Infrastructure/Migrations/ChirpDBContextModelSnapshot.cs

```diff
@@ -17,33 +17,19 @@ protected override void BuildModel(ModelBuilder modelBuilder)
         {
 #pragma warning disable 612, 618
             modelBuilder
-                .HasAnnotation("ProductVersion", "7.0.13")
+                .HasAnnotation("ProductVersion", "8.0.0")
```

Comment:
Does this refer to our .NET version? 'Cause then it needs to be reverted to 7.0.

Reply:
Ah good eye, it was indeed that. Fixed now.